WordPress/Sniffs/NamingConventions/ValidVariableNameSniff.php

```diff
@@ -57,6 +57,15 @@ class WordPress_Sniffs_NamingConventions_ValidVariableNameSniff extends PHP_Code
 	);
 
 	/**
+	 * Mixed-case variables used by WordPress.
+	 *
+	 * @var array
+	 */
+	public $wordpress_mixed_case_vars = array(
+		'EZSQL_ERROR',
```

Comment:
I was just following the pattern already set with the sniff's other similar properties. Obviously, using `protected` and `isset()` is better, though. Changing the other properties would break back-compat, so I haven't modified them.